clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -270,10 +275,14 @@ func (p *podStartupLatencyMeasurement) gather(c clientset.Interface, identifier
 //  it just returns incomplete results.
 //  Given that we don't 100% accuracy, we should switch to a mechanism that is similar
 //  to the one that slo-monitor is using (added in #1477).
-func (p *podStartupLatencyMeasurement) gatherScheduleTimes(c clientset.Interface) error {
+func (p *podStartupLatencyMeasurement) gatherScheduleTimes(c clientset.Interface, schedulerName string) error {
+	scheduler := corev1.DefaultSchedulerName
+	if len(schedulerName) != 0 {
+		scheduler = schedulerName
+	}
```

Comment:
We don't needs this anymore - we already have default value above if someone does not specify schedulerName

Reply:
Agree, removed it.